test/test_consumer_integration.py

```diff
@@ -88,6 +94,12 @@ def kafka_consumer(self, **configs):
                                  **configs)
         return consumer
 
+    def kafka_producer(self, **configs):
+        brokers = '%s:%d' % (self.server.host, self.server.port)
+        producer = KafkaProducer(
+            bootstrap_servers=brokers, **configs)
+        return producer
```

Comment:
Sadly we can't use pytest fixtures here, as it's a `TestCase` class. We will need to refactor those classes to simple functions to support `fixtures`.